src/js/navIcons.js

```diff
@@ -53,6 +53,9 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
      */
     fluid.defaults("gpii.firstDiscovery.navIcons", {
         gradeNames: ["fluid.viewRelayComponent", "autoInit"],
+        pageSize: 5,
+        iconWidth: "6rem",
+        iconHoles: [2, 8], // a list of all the panel positions which have no nav icons (currently the "welcome" and "congratulations" pages)
```

Comment:
With 8 being added to the list, you probably wanna add a placeholder icon in index.html for the congratulation page like what has been done for the welcome page, or, remove 8 from this list. I lean towards the former as the placeholder icon indicates an associated panel.